docs/src/content/docs/api/calls.mdx

```diff
@@ -147,49 +147,59 @@ An optional query parameter called `priorCallId` can be provided to continue a p
         <th class="w-10/12"></th>
       </tr>
       <tr>
-        <td class="font-mono">systemPrompt <br /> <span class="text-sm text-red-500">required</span></td>
+        <td class="font-mono">initator</td>
         <td>string</td>
-        <td>The system prompt to use for the AI.</td>
+        <td>Who was responsible for staring this call. Typically set to agent for outgoing calls and left as the default (user) otherwise.<br />Can be `"agent"` or `"user"` (default).</td>
       </tr>
       <tr>
         <td class="font-mono">languageHint</td>
         <td>string</td>
         <td>A hint about the language to be used if your end users are speaking a language other than English. Must be a valid BCP47 language code. Best effort is made to select the closest supported language.  (default: "en")</td>
       </tr>
+      <tr>
+        <td class="font-mono">maxDuration</td>
+        <td>string</td>
+        <td>Maximum length for the call in the format `HH:MM:SS`. Can be used to limit the length of the call. Optionally, set `timeExceededMessage` to have the agent say a message when the allotted time is reached.<br /><br />For the free plan, defaults to amount of free time remaining (less any calls that are currently in-progress). For paid plans, defaults to the system maximum of one hour ("01:00:00"). </td>
```

Comment:
Please update format.  Should be number of seconds (possibly fractional) followed by "s".

Reply:
done